src/style.css

```
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts\Avenir-Book.ttf) format("truetype");
+  font-style: initial;
+  font-weight: initial;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(fonts\Avenir-Heavy.ttf) format("truetype");
+  font-style: normal;
+}
+
+form {
+  margin-top: 20px;
+}
+
+.first__field {
+  width: 100%;
+  height: 70px;
+  font-weight: 300;
```

```suggestion
  width: 100%;
  height: 70px;
  font-weight: 300;
```
ці стилі дублюються для обох інпутів. Створи загальний клас для загальних стилів, і два окремих стиля для кожного інпута окремо, а потім використовуй їх на зразок 'input input__large' та 'input input__small'